### PR TITLE
Milestone: IPET Evaluation

### PR DESCRIPTION
**Pull request by fschloesser**
- clean up code
- update search
- implement ipet evaluation
- reimport testsets via ipet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Rubberband
 
-A flexible LP solver log file web view and analysis tool, backed by Elasticsearch.
+A flexible LP solver log file web view and analysis tool, backed by Elasticsearch!
```
Comment (ambros-gleixner) [inline README.md]: A flexible web view and analysis platform for solver log files of mathematical optimization software, backed by Elasticsearch!

Comment (GregorCH) [reply README.md]: backed by Elasticsearch.

**Comment by ambros-gleixner** [comment]
As discussed feel free to merge. We will organize a general code review of Rubberband in the next weeks.

**Comment by fschloesser** [comment]
I would like to merge this, as this is the current up-to-date stable version.

**Comment by ambros-gleixner** [comment]
Of course!

**Pull request merged**